src/config/tsconfig.aegir.json

```
@@ -31,7 +31,7 @@
         "forceConsistentCasingInFileNames": true,
         "skipLibCheck": true,
         "stripInternal": true,
-        "resolveJsonModule": true
+        "resolveJsonModule": true,
```

Do we need to lint JSON as well?  Edit for consistency:

```suggestion
        "resolveJsonModule": true
```

---

Might be a good idea, we can create a separate issue for it.